packages/public-api/src/clientApiRouter.ts

```diff
@@ -45,6 +45,7 @@ type RequestWebPageScanResponse = {
     identifiedModuleMap: Record<string, WebPageScan.IdentifiedModule>;
     identifiedPackages: ScanResultPackageWithMetadata[];
     vulnerabilities: Record<string, PackageVulnerabilityData[]>;
+    processedScripts: WebPageScan.ProcessedScript[];
```

Comment:
Can you please also add this field on the private backend? An identified bundler can be extracted from the bootstrap script